tests/virt_autotest/libvirt_extend_storage_lvm.pm

```diff
@@ -0,0 +1,143 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Extend LVM Storage pool / volume test
+#
+# Test flow:
+# - Check a additional free hard disk
+# - Wipe hard disk clean
+# - Partition test disk
+# - Create a pv and display result
+# - Create a vg named 'lvm_vg' and display result
+# - Create a storage pool (lv) named 'guest_image_lvm'
+# - Create a logical volume (lv) size 1G
+# - Attach a logical volume (lv) to guest systems
+# - Clone a logical volume (lv)
+# - Cleanup
+# Maintainer: Leon Guo <xguo@suse.com>
+
+use base "virt_feature_test_base";
+use virt_autotest::utils;
+use virt_autotest::common;
+use strict;
+use warnings;
+use testapi;
+use utils;
+use virt_utils;
+use version_utils 'is_sle';
+
+sub run_test {
+    my ($self) = @_;
+
+    record_info "Prepare Guest Systems";
+    foreach (keys %virt_autotest::common::guests) {
+        start_guests() unless is_guest_online($_);
+    }
+
+    ## About LVM volumes management
+    my ($sys_disk, $dev, $lvm_disk);
+    # Get current system disk name
+    $sys_disk = script_output 'lsblk -rnoPKNAME $(findmnt -nrvoSOURCE /)';
+    # Check with all existed Hard disks
+    my @disks = split(/\n/, script_output("lsblk -n -l -o NAME -d -e 7,11"));
+    # Use a additional free hard disk for LVM volumes
+    $dev = "/dev/";
+    if ($sys_disk eq $disks[0]) {
+        $lvm_disk = $dev . $disks[2];
```

Comment:
why is $disks[2] choosen to be used?

Reply:
currently, refer to virt_autotest/reset_partition, which would be used $disks[1]. 
Depend on this situation, so use with $disks[2] here.

Reply:
Same as above, if the machine has $disk[2] should be considered as well.

Reply:
yes, try to update here.

Reply:
had update code to find out unused hard disk